oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
@nathanielmanistaatgoogle After [reading][1] a bit more, I don't think it is worth it given the existing tooling.

[1]: http://stackoverflow.com/questions/8100166/inheriting-methods-docstrings-in-python